### PR TITLE
Cassandane::Tiny: Import strict/warnings to consumers

### PR DESCRIPTION
**Pull request by wolfsage**
This way any tiny test gets them by default, avoiding mistakes like forgetting to declare variables, etc...

### CODE REVIEW

**Comment by wolfsage** [comment]
(This identified a bunch of compilation problems that need to be cleaned up; I'll work on that and include them in this MR...)

**Comment by wolfsage** [comment]
This should be all set

**Review by rsto — APPROVED** [review]
Woof - nice!

**Comment by rsto** [comment]
Waiting for @elliefm to chime in, too.

**Comment by rjbs** [comment]
Wow, thanks.

File: cassandane/tiny-tests/JMAPCalendars/calendar_set_defaultalerts_shared [hunk 1]
```diff
@@ -134,10 +137,10 @@ sub test_calendar_set_defaultalerts_shared
         }, 'R1'],
     ]);
 
-    $self->assert_shared_defaultalerts($t);
+    $self->assert_shared_defaultalerts($t, 'no alarm!', 'no alarm!');
 }
 
-sub _can_match {
+sub _can_match_b {
```
Comment (elliefm) [inline cassandane/tiny-tests/JMAPCalendars/calendar_set_defaultalerts_shared:143]: Are these two functions identical?  In that case I would've just moved it up to the parent.  The fact that you gave them distinct names instead makes me think maybe they were distinct functions sharing a name, but the bodies don't show in the diff and I didn't look any deeper to find out

**Review by elliefm — APPROVED** [review]
Nice

**Review by ksmurchison — APPROVED** [review]

**Review by rjbs — APPROVED** [review]

File: cassandane/tiny-tests/JMAPCalendars/calendar_set_defaultalerts_shared (hunk 1 above)
Comment (rjbs) [reply cassandane/tiny-tests/JMAPCalendars/calendar_set_defaultalerts_shared:143]: They are identical and also appear in cassandane/Cassandane/Cyrus/CaldavAlarm.pm

**Comment by wolfsage** [comment]
Hey thanks.

**Pull request merged**